frontend/src/lib-components/layout/Container.tsx

```diff
@@ -109,10 +116,20 @@ export const CollapsibleContentArea = React.memo(
           role="button"
         >
           {title}
-          <TitleIcon
-            icon={open ? faChevronUp : faChevronDown}
-            data-qa="collapsible-trigger"
-          />
+          <IconContainer>
+            {countIndicator > 0 && (
+              <>
+                <CircledChar color="white" data-qa="count-indicator">
```

Comment:
We already have a `RoundIcon` component that could be used here. 

Also, citizen application sections have error indicators that basically this same thing. You could use this same mechanism there if you want to do the refactoring.

Reply:
Excellent idea. To be compatible with the underlying roundicon implementation had to use readableColor in icon color and thus it is now black instead of the UX specified white but according to discussion this is as intended